docs/installation/_index.md

```diff
@@ -19,29 +19,18 @@ aliases:
     - /getting-started/linux-install/
 ---
 
-To use Viam software with your robot, you need to install and run a binary called `viam-server` on the computer you want to use to control the robot.
-See [What is Viam?](/viam/) for more information.
+To use Viam software with your robot, you need to install and run the `viam-server` binary on the computer that you want to use to control the robot.
+In most cases, this will be a [single board computer (SBC)](#install-on-a-single-board-computer), like a Raspberry Pi, but you can also use a [macOS or Linux computer](#install-on-a-macos-or-linux-computer) to evaluate `viam-server` in development.
 
-## Preparation
+For an overview of Viam software, see [Viam in 3 minutes](/viam/).
 
-Make sure your system is compatible with Viam.
-You can run `viam-server` on any device that runs:
+## Install on a single board computer
 
-- **any** Linux 64-bit operating system (except FUSE)
-- macOS
+### Prepare your board
 
-<div class="container text-left td-max-width-on-larger-screens">
-    <div class="row">
-        <img src="img/thumbnails/pc.png" style="max-width:100px" alt="Desktop computer">
-        <div class="col">
-            <p> If you plan to run <code>viam-server</code> on your laptop or desktop with a Linux or Mac operating system, no special prep is required. Proceed to <a href="#install-viam-server">Install <code>viam-server</code></a>. </p>
-        </div>
-    </div>
-</div>
-
-<br>
-
-If you are using a single board computer (SBC) or a microcontroller, prepare your device by following the relevant setup document:
+If you haven't already, you must install a supported operating system to your {{< glossary_tooltip term_id="board" text="board" >}}.
+`viam-server` supports **any** Linux 64-bit operating system (except FUSE).
```

Comment:
should except FUSE be in parentheses or not? I think you know more about that than me so up to you what you think is best

Reply:
Right on, this has been slightly bothering me for a bit: FUSE is not an OS itself, but available for most OSes. I'll take this opportunity to find the right wording for this once and for all -- thanks!

Reply:
Spoke with James, updated copy with several clarifications surrounding OS and  arch support: such as the fact that we actually do need FUSE2 or later on Linux for the AppImage approach!